dist/s_string

```diff
@@ -33,7 +33,8 @@ check() {
 	# Strip out git hashes, which are seven character hex strings.
 	# Strip out double quote char literals ('"'), they confuse aspell.
 	# Strip out calls to __wt_getopt so the option lists don't have to be spelling words.
-	sed -e 's/ [0-9a-f]\{7\} / /g' -e "s/'\"'//g" -e 's/__wt_getopt([^()]*)//' ../$2 |
+	# Strip out C-style hex constants.
+	sed -e 's/ [0-9a-f]\{7\} / /g' -e "s/'\"'//g" -e 's/__wt_getopt([^()]*)//' ../$2 -e 's/0x[[:xdigit:]]\+//' |
```

Comment:
Firstly, I was deliberately attempting to leave existing expressions unchanged if I could. The new expression I added also has '0x' at the start, so it will (should) only match C/C++ style hex constants which have a '0x' at the start. 

Secondly, without the '0x', mistyped words could also match as hex (eg typing 'af' instead of 'at'). The pre-existing expression only matches 7 character hex values (and so would match 'af'), whereas mine would match to any length. 

I think this means the code is correct.